test/test_mirror.sh.in

```diff
@@ -22,7 +22,10 @@
 nerrors=0
 
 SERVER_VERBOSITY="--verbosity=1"
-SERVER_PORT="--port=3000"
+# Choose random ephemeral port number
```

Comment:
Do not hardcode port #, especially one below 49152, which are allocated to existing services.   Randomizing the port # helps avoid choosing the same port # as a concurrency running mirror test script, albeit imperfectly.